### PR TITLE
feat: add token revoke flow

### PR DESCRIPTION
**Pull request by lcpojr**
Add a new command for revoking access and refresh tokens.
In order to make this work i had to do some minor changes on all sign in flows for start creating refresh token sessions.

### CODE REVIEW

File: apps/authenticator/lib/sign_out/commands/inputs/revoke_tokens.ex [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  @moduledoc """
+  Input schema to be used in Revoke Tokens flow.
+  """
+
+  use Authenticator.Input
+
+  @typedoc "Revoke Tokens flow input fields"
+  @type t :: %__MODULE__{
+          access_token: String.t() | nil,
+          refresh_token: String.t() | nil
```
Comment (dcdourado) [inline apps/authenticator/lib/sign_out/commands/inputs/revoke_tokens.ex:11]:
```suggestion
          access_token: String.t(),
          refresh_token: String.t()
```

These fields are required

File: apps/authenticator/lib/sign_out/commands/inputs/revoke_tokens.ex [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+  @required [:access_token, :refresh_token]
+  embedded_schema do
+    field :access_token, :string
+    field :refresh_token, :string
+  end
+
+  @doc false
+  def changeset(params) when is_map(params) do
+    %__MODULE__{}
+    |> cast(params, @required ++ @optional)
```
Comment (yashin5) [inline apps/authenticator/lib/sign_out/commands/inputs/revoke_tokens.ex:23]: this attribute module does not exist

```suggestion
    |> cast(params, @required)
```

File: apps/authenticator/lib/sign_out/commands/inputs/revoke_tokens.ex (hunk 1 above)
Comment (lcpojr) [reply apps/authenticator/lib/sign_out/commands/inputs/revoke_tokens.ex:11]: Actually it's not mandatory anymore.

File: apps/authenticator/lib/sign_out/commands/inputs/revoke_tokens.ex (hunk 2 above)
Comment (lcpojr) [reply apps/authenticator/lib/sign_out/commands/inputs/revoke_tokens.ex]: Actually it has optionals now.

**Pull request merged**